hedera-node/docs/system-accounts-operations.md

```diff
@@ -0,0 +1,187 @@
+| 0x0                                | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns 0 as if account doesn’t exist                                       |
+| 0x1 → 0x9                          | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns 0 as if account doesn’t exist                                       |
+| 0xA → 0x166                        | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns 0 as if account doesn’t exist                                       |
+| 0x167 (0.0.359)                    | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns 0 as if account doesn’t exist                                       |
+| 0x168, 0x169 (0.0.360, 0.0.361)    | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns 0 as if account doesn’t exist                                       |
+| 0x16a → 0x2EE (0.0.361 → 0.0.750)  | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns 0 as if account doesn’t exist                                       |
+| 0x2EF → 0x3E8 (0.0.751 → 0.0.1000) | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns balance or 0 if an account or contract doesn’t exist on the address |
+| 0x3E8 → ♾️                         | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns balance or 0 if an account or contract doesn’t exist on the address |
+
+### SelfDestruct operation
+
+**Ethereum:** 
+    - `SelfDestruct` executed in the same transaction that the contract was created - the contract data is deleted (storage keys, code, etc.), the account balance is transferred to the target. If the target is the same as the contract calling `SelfDestruct`, the Ether will be burnt.
+    - otherwise - all funds are recovered but any other account data is not deleted. The entire account balance is transferred to the target. If the target is the same as the contract calling `SelfDestruct`, there is no net change in the balances. Ether will not be burnt in this case.
+
+**Hedera:**
+1. For addresses in range 0x0 → 0x2EE (0.0.0 → 0.0.750): fail with status `INVALID_SOLIDITY_ADDRESS`.
+2. For addresses in range 0x2EF → 0x3E8 (0.0.751 → 0.0.1000):
+   - success, if account exists and has `receiverSigRequired` == false;
+   - success, if account exists and has `receiverSigRequired` == true and account is `sender`;
+   - fail in some failing scenarios such as: 
+     - beneficiary same as account-to-be-deleted - fail with status `SELF_DESTRUCT_TO_SELF`;
+     - contract-to-be-deleted is a token treasury - fail with status `CONTRACT_IS_TREASURY`;
+     - contract-to-be-deleted has a positive balance - fail with status `TRANSACTION_REQUIRES_ZERO_TOKEN_BALANCES`;
```

Comment:
Probably we can elaborate more on this case, e.g.:

contract-to-be-deleted has a positive `fungible token` balance, 

so that we are not misinterpreting this for hbar balance.